Library/Formula/kwm.rb

```diff
@@ -0,0 +1,58 @@
+class Kwm < Formula
+  desc "Tiling window manager with focus follows mouse for OSX."
+  homepage "https://koekeishiya.github.io/kwm/"
+  url "https://github.com/koekeishiya/kwm/archive/v1.0.0-RC2.tar.gz"
+  version "1.0.0RC2"
+  sha256 "61252c4710c189cd7f101a3f2555bfdfb2c58c0e32b7856b51bf4e0ba0579202"
+
+  def install
+    system "make", "install"
+    bin.install "bin/kwm"
+    bin.install "bin/kwmc"
```

Comment:
What is `make install` even doing here, given it's not being given a prefix or anything?